src/Resources/config/services.xml

```diff
@@ -13,5 +13,15 @@
                  class="Paysera\LoggingExtraBundle\Service\CorrelationIdProvider">
             <argument>%paysera_logging_extra.application_name%</argument>
         </service>
+
+        <service id="paysera_logging_extra.sentry_handler"
```

Comment:
Let's move this definition under `services/handlers.xml` and rename it to `paysera_logging_extra.handler.sentry`. Then we can define an alias here called `paysera_logging_extra.sentry_handler`, which points to the concrete handler.